contracts/tonomy/include/tonomy/tonomy.hpp

```diff
@@ -248,6 +273,9 @@ namespace tonomysystem
          string description;
          string logo_url;
          string origin;
+         string background_color;
```

Comment:
```c++
struct [[eosio::table]] app
      {
         name account_name;
         string json_data; // json string with app name, desciption, logo URL, and colours. We use the version number to know how to parse the json
         uint16_t version;
         checksum256 username_hash;
         string origin;
```

Reply:
we cannot just change a table in-place. we will need to create a new structure and table, and migrate all apps to the new table. so this will need a migration function